.github/workflows/avm.ptn.data.private-analytical-workspace.yml

```diff
@@ -22,6 +22,7 @@ on:
         type: string
         description: "Default location overwrite (e.g., eastus)"
         required: false
+        default: 'northeurope'
```

Comment:
Please don't hardcode the value in the workflow (which overwrites the location in all tests), but rather replace the default resource location parameter 
```bicep
@description('Optional. The location to deploy resources to.')
param resourceLocation string = deployment().location
```
with 
```bicep
// enforcing location due to quote restrictions
#disable-next-line no-hardcoded-location
var enforcedLocation = 'northeurope'
```
in the `main.test.bicep` files and update the references. This allows you/us to specify different locations for different tests and is more in line with other, similar cases like in this module

https://github.com/Azure/bicep-registry-modules/blob/a0c516bd1d7b34f4a09278df077a25f15a89a2c6/avm/res/net-app/net-app-account/tests/e2e/defaults/main.test.bicep#L14-L16